### DISCUSSION TITLE
Change the position of the custom tooltip based on screen size

### DISCUSSION THREAD

**Discussion by asifkabani** [gilbarbara/react-joyride#931] [Q&A]
Is there a way to handle responsive behavior, meaning that when the tooltip is showing, if I adjust the window size to be smaller, for it to detect that and reposition the tooltip? Currently when I try to do that it does not seem to work.

**Comment by gilbarbara** [comment]
@asifkabani 
Have you tried changing the step's `placement` to **auto**?
